src/CreateGoalPanel1/index.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import classNames from 'classnames';
+import SectionHeaderGeneral from '../SectionHeaderGeneral';
+import GoalMenuItem from '../GoalMenuItem';
+import Spacer  from '../Spacer';
+import styles from './styles.css';
+import SectionFooterNew from '../SectionFooterNew';
+
+
+function CreateGoalPanel1(props){
+
+      return (
+
+
+        <div className={styles.outerWrapper}>
+              <SectionHeaderGeneral
+                      onClick={props.onCancel}
```

Comment:
onCancel isn't part of your props validation (the propTypes object at the end of your component). The component works fine without it however having a propType ensures the component is used as intended. You can read more [here](https://facebook.github.io/react/docs/reusable-components.html)